### PR TITLE
Made get endpoint

### PR DESCRIPTION
**Pull request by Preston-Shipman**
products

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please add and commit your server changes too.

File: server/index.js [hunk 1]
```diff
@@ -19,8 +19,21 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
-app.use('/api', (req, res, next) => {
-  next(new ClientError(`cannot ${req.method} ${req.originalUrl}`, 404));
```
Comment (thebearingedge) [inline server/index.js:23]: You didn't need to delete this middleware. It's there for JSON requests to endpoints that don't exist.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 1 above)
Comment (Preston-Shipman) [reply server/index.js:23]: Oh ok ill add it back in

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, almost there. Your `app.get('/api/products', etc etc` should come right after the one for `/api/health-check`

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**